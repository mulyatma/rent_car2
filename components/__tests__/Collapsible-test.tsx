import * as React from 'react';
import renderer from 'react-test-renderer';

import { Collapsible } from '../Collapsible';

it(`renders correctly`, () => {
  const tree = renderer.create(<Collapsible>Snapshot test!</Collapsible>).toJSON();

  expect(tree).toMatchSnapshot();
});
