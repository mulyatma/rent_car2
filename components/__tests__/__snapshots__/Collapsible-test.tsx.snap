// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      undefined,
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "gap": 6,
        "opacity": 1,
      }
    }
  >
    <ViewManagerAdapter_SymbolModule
      animated={false}
      colors={[]}
      name="chevron.right"
      resizeMode="scaleAspectFit"
      style={
        [
          {
            "height": 24,
            "width": 24,
          },
          [
            {
              "height": 18,
              "width": 18,
            },
            {
              "transform": [
                {
                  "rotate": "0deg",
                },
              ],
            },
          ],
        ]
      }
      tintColor="#687076"
      type="monochrome"
      weight="medium"
    />
    <Text
      style={
        [
          {
            "color": "#11181C",
          },
          undefined,
          undefined,
          {
            "fontSize": 16,
            "fontWeight": "600",
            "lineHeight": 24,
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    />
  </View>
</View>
`;
